readme.md

```diff
@@ -1,7 +1,7 @@
 # Stars block
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://przwojwwp.github.io/layout_stars/)
+- [TEST REPORT LINK](https://przwojwwp.github.io/layout_stars/report/html_report/)
```

Comment:
add this links to PR description 
![image](https://github.com/mate-academy/layout_stars/assets/70009205/b9c5b267-a1d9-4e3f-9064-169151622e42)